lib/opentelemetry-proto/src/convert.rs

```diff
@@ -1,45 +1,47 @@
-use super::proto::{
-    common::v1::{any_value::Value as PBValue, KeyValue},
-    logs::v1::{LogRecord, ResourceLogs, SeverityNumber},
-    resource::v1::Resource,
-};
 use bytes::Bytes;
-use chrono::{TimeZone, Utc};
+use chrono::{DateTime, TimeZone, Utc};
+use lookup::path;
 use ordered_float::NotNan;
 use std::collections::BTreeMap;
 use value::Value;
 use vector_core::{
-    config::log_schema,
+    config::{log_schema, LegacyKey, LogNamespace},
     event::{Event, LogEvent},
 };
 
-const RESOURCE_KEY: &str = "resources";
-const ATTRIBUTES_KEY: &str = "attributes";
-const TRACE_ID_KEY: &str = "trace_id";
-const SPAN_ID_KEY: &str = "span_id";
-const SEVERITY_TEXT_KEY: &str = "severity_text";
-const SEVERITY_NUMBER_KEY: &str = "severity_number";
-const OBSERVED_TIMESTAMP_KEY: &str = "observed_timestamp";
-const DROPPED_ATTRIBUTES_COUNT_KEY: &str = "dropped_attributes_count";
-const FLAGS_KEY: &str = "flags";
+use super::proto::{
+    common::v1::{any_value::Value as PBValue, KeyValue},
+    logs::v1::{LogRecord, ResourceLogs, SeverityNumber},
+    resource::v1::Resource,
+};
+
+const SOURCE_NAME: &str = "opentelemetry";
+
+pub const RESOURCE_KEY: &str = "resources";
+pub const ATTRIBUTES_KEY: &str = "attributes";
+pub const TRACE_ID_KEY: &str = "trace_id";
+pub const SPAN_ID_KEY: &str = "span_id";
+pub const SEVERITY_TEXT_KEY: &str = "severity_text";
+pub const SEVERITY_NUMBER_KEY: &str = "severity_number";
+pub const OBSERVED_TIMESTAMP_KEY: &str = "observed_timestamp";
+pub const DROPPED_ATTRIBUTES_COUNT_KEY: &str = "dropped_attributes_count";
+pub const FLAGS_KEY: &str = "flags";
 
-impl IntoIterator for ResourceLogs {
-    type Item = Event;
-    type IntoIter = std::vec::IntoIter<Self::Item>;
-    fn into_iter(self) -> Self::IntoIter {
+impl ResourceLogs {
+    pub fn into_iter(self, log_namespace: LogNamespace) -> impl Iterator<Item = Event> {
```

Comment:
Same with this one. Could this be `into_event_iter`?

Reply:
Resolved in https://github.com/vectordotdev/vector/pull/15469/commits/b424739780a0b45ef32d2bd51570537c1391bca7